src/main/scala/viper/gobra/Gobra.scala

```diff
@@ -82,9 +89,18 @@ trait GoVerifier extends StrictLogging {
       }
     })
 
+    val viperBackendName: String = config.backend match {
+      case CarbonBackend => "Carbon"
+      case SiliconBackend => "Silicon"
+    }
+
     val timeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss");
     config.packageInfoInputMap.keys.foreach(pkgInfo => {
       val pkgId = pkgInfo.id
+
+      //adds a submitter for this package to submitters if config.submitForEvaluation is set, key is pkgId
```

Comment:
this sounds like the wrong place for this comment. Sth like this should be in `createSubmitter` and and the description of what we store in keys/values should be on the declaration of `submitters`